indico/modules/events/abstracts/schemas.py

```diff
@@ -69,6 +72,15 @@ class Meta:
                   'address', 'phone', 'is_speaker', 'author_type')
 
 
+class AbstractFileSchema(mm.SQLAlchemyAutoSchema):
+    icon = mm.Function(lambda af: icon_from_mimetype(af.content_type))
+    download_url = mm.Function(lambda af: url_for('abstracts.download_attachment', af))
```

Comment:
Would it make sense to use an external URL here?

Reply:
hm, we don't do so either for paper files... but yeah, might be nice to change it for all places where it's an export that's meant to be downloaded and not used in indico itself